lighthouse/lighthouserc.js

```diff
@@ -59,6 +61,23 @@ const AUDIT_URLS = {
       'https://www.test.bbc.com/pashto/afghanistan-52643309?renderer_env=live',
     ],
   },
+
+  TOPIC_PAGE: {
+    live: ['https://www.bbc.com/pidgin/topics/c0823e52dd0t'],
+    test: ['https://www.test.bbc.com/pidgin/topics/c0823e52dd0t'],
+  },
+
+  HOME_PAGE: {
```

Comment:
Just wondering about the naming, we have 'Front page' and now 'Home page'? Could we use a better name?

Reply:
This is temporary - because we have 2 separate page types, so we need to be able to distinguish between them until we have migrated all services over the new HomePage. There will be a later PR where we either delete all references to FrontPage, and keep HomePage, or we remove HomePage and rename everything to FrontPage.

Reply:
TIPO_HOME_PAGE?